README.md

```diff
@@ -32,8 +32,8 @@ Once the validation is successful, you can commit & push your pool config.
 
 ## 5. Github Pull Request
 
-clic on this button to create a Pull Request (from your **forked v5 branch** TO **iexec-pools-registry v5 branch**):
+click on this button to create a Pull Request (from your **forked v5 branch** TO **iexec-pools-registry v5 branch**):
```

Comment:
V5 sould be replaced by "latest release branch" ?

Reply:
ty fixed 94298eb11d043cd5ef2df7c0fe0150aec439f56c